ui/plugins/controller.js

```diff
@@ -439,6 +439,7 @@ treeherder.controller('PluginCtrl', [
 
         var selectJobAndRender = function(job) {
             $scope.jobLoadedPromise = selectJob(job);
+            $( "#info-panel" ).addClass( "info-panel-slide" );
```

Comment:
I see the jQuery docs also include white space in their paren () examples
https://api.jquery.com/addclass

Anyway, @wlach let me know which you think we should do across the treeherder repo, I will make it so.